runner/src/server/forms/test.json

```diff
@@ -481,7 +481,7 @@
       "title": "Update form in application store",
       "type": "savePerPage",
       "outputConfiguration": {
-        "savePerPageUrl": "https://webhook.site/ba83e49b-cdd1-46e9-9c04-c89cfa651752"
+        "savePerPageUrl": "True"
```

Comment:
Was this changed on purpose to a string?  Is the "True" just for a truthy evaluation?